tools/CoqMakefile.in

```diff
@@ -1,4 +1,4 @@
-###############################################################################
+##############################################################################
```

Comment:
Probably an editing mistake. Will correct that, since it seems so crucial for you...